Tests/BCCryptoTestsRelease.xcconfig

```diff
@@ -1,2 +1,4 @@
 #include "../BCFoundation/Configs/BCTestsRelease.xcconfig"
 #include "BCCryptoTests.xcconfig"
+
+LLVM_LTO = NO
```

Comment:
This came about during #3977. I realised the BCCrypto tests weren't being run. When I enabled them we got a crash somewhere in the build chain for release targets. I twiddled the build flags until the tests ran again.